src/ratgdo.h

```diff
@@ -64,6 +64,7 @@ struct GarageDoor {
     bool light;
     LockCurrentState current_lock;
     LockTargetState target_lock;
+    long unsigned int led_on_time;
```

Comment:
since this isn't a property of the garage door, I'd prefer it live at the top level (i.e. as a free variable in ratgdo.cpp)

Reply:
LoL.... he had it defined as an independent global with externs.  I suggested moving it here though I was wondering if that was wrong based on it not actually being part of the garage door.